sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/expressions.scala

```diff
@@ -449,7 +452,7 @@ object FoldablePropagation extends Rule[LogicalPlan] {
           stop = true
           j
 
-        // These 3 operators take attributes as constructor parameters, and these attributes
+        // These 4 operators take attributes as constructor parameters, and these attributes
```

Comment:
shall we turn to whitelist? As I remember there have been more than 3 bugs reported on this optimizer rule.